src/ImageSharp/ImageSharp.csproj

```diff
@@ -28,6 +28,14 @@
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
   </ItemGroup>
 
+  <!-- Enable support for using libgdiplus on macOS -->
```

Comment:
This isn't a requirement for the main library. ImageSharp does not use System.Drawing